tests/e2e/vLLM/test_vllm.py

```diff
@@ -130,7 +130,6 @@ def test_vllm(self):
         session.reset()
 
         if SKIP_HF_UPLOAD.lower() != "yes":
-
```

Comment:
our formatters must be different if you're formatting this file i recently updated. are you doing `make style`? Maybe i forgot to run it and it still passed style checks?

Reply:
I'm running ruff=0.4.10